p2p/libp2p/netMessenger_test.go

```diff
@@ -1693,6 +1693,48 @@ func TestNetworkMessenger_ApplyOptionsShouldWork(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestNetworkMessenger_ChooseAnotherPortIfBindFails(t *testing.T) {
+	port := "37000-37100" //use a larger interval as some ports might already be used by the OS
```

Comment:
skip this test by default? it might fail or take long times on some machines

Reply:
right, will be disabled by default

Reply:
done